checker/src/org/checkerframework/checker/units/UnitsRelationsTools.java

```diff
@@ -178,17 +207,42 @@ private static boolean hasNoPrefix(/*@Nullable*/ final AnnotationValue annotatio
     }
 
     /**
-     * Removes the Prefix value from an Annotation, by constructing and returning a copy of its base SI unit's Annotation
-     * @param elements
-     *            Element Utilities from a checker's processing environment,
-     *            typically obtained by calling env.getElementUtils() in init()
-     *            of a Units Relations implementation
-     * @param unitsAnnotation An AnnotationMirror representing a Units Annotation
-     * @return The base SI Unit's AnnotationMirror, or null if the base SI Unit cannot
-     *         be constructed
+     * Given an Annotated Type, this method returns the Annotation representing
+     * the unit that this Annotated Type has.
+     *
+     * @param annoType an Annotated Type
+     * @return an AnnotationMirror representing the unit that this Annotated
+     *         Type has
+     */
+    public static /*@Nullable*/ AnnotationMirror getUnit(/*@Nullable*/ final AnnotatedTypeMirror annoType) {
+        if (annoType == null) {
+            return null;
+        }
+        // return the first unit (should always be the only unit) in the
```

Comment:
Why do you need this??
For certain units, don't you use multiple annotations, e.g. in `@m @s`?